endpoints/functions.php

```diff
@@ -1123,19 +1123,11 @@ function getIvrResponse($redirected_from = null, $prior_digit = null, $expected_
 }
 
 function getInputType() {
-    if (has_setting('speech_gathering') && json_decode(setting('speech_gathering'))) {
-        $inputType = "speech dtmf";
-    } else {
-        $inputType = "dtmf";
-    }
+    $inputType = has_setting('speech_gathering') && json_decode(setting('speech_gathering')) ? "speech dtmf" : "dtmf";
```

Comment:
You can just return rather than assign to a new variable and return.

Reply:
Cool wasnt sure about that, makes sense.